contribution_process.md

```diff
@@ -0,0 +1,164 @@
+
+1. Changes to build processes. Some of our build processes are available through GitHub, but some are not. We need to carefully test any potential changes to be sure that they are compatible with all of our systems.
+1. GitHub Actions. Changes to our actions need to be made and reviewed with extra scrutiny for security purposes.
+1. Changes to models, including waiters, paginators, and resources. These requests are cross-SDK and managed by service teams so that all SDKs can use them with uniform behavior.
+1. Changes to configuration or credential files or processes. These changes can also affect the behavior in other SDKs and must thus be made in coordination with internal teams.
+
+## Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer adds the
+issue to the "Ready to contribute" lane of the public [GitHub project
+board](https://github.com/orgs/aws/projects/21). This stage indicates any user
+is welcome to initiate a contribution via a GitHub pull request.
+
+## Implementation stage
+
+A GitHub issue enters the implementation stage once a user submits a GitHub pull
+request implementing the change. This stage indicates that a contribution has
+been submitted for the issue, but the maintainers have not committed to a full
+review of the contribution. The issue will be moved to the "Implementation" lane
+of the [GitHub project board](https://github.com/orgs/aws/projects/21). The
+maintainers will respond to let the contributor know to expect a preliminary
+review to confirm:
+
+1. It is implemented in the manner described in the issue.
+2. It meets the following  criteria:
+   - The pull request must include passing tests and must pass all existing tests.
+   - The code must work on all supported operating systems.
+   - The code must be documented, which may include inline documentation and user documentation.
+   - A CHANGELOG entry must be provided.
+
+If the criteria are not satisfied, the maintainers will comment on the issue
+specifically indicating what work is still required. If the contributor has
+questions, they should refer to this guide or ask the maintainers to clarify.
+Once the contributor makes the required changes, the maintainers will follow up.
+
+If the pull request is still a work in progress, contributors should open it as a [draft pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request). A preliminary review will not be made on pull requests with a draft status. When the contributor is ready for review, they should mark it as ready (remove draft status). The contributor should feel free to ask questions or request help if their pull request is in the draft status.
```

Comment:
Let's fix the line wrap length on this as well to match the rest of document as well.

Reply:
done with reflow of whole doc